src/lib.rs

```diff
@@ -175,15 +208,23 @@ pub fn store<T: Serialize>(name: &str, cfg: T) -> Result<(), ConfyError> {
 
     let config_dir_str = get_configuration_directory_str(&project)?;
 
-    let path: PathBuf = [config_dir_str, &format!("{}.toml", name)].iter().collect();
+    let path: PathBuf = [config_dir_str, &format!("{}.{}", name, EXTENSION)].iter().collect();
 
     let mut f = OpenOptions::new()
         .write(true)
         .create(true)
         .truncate(true)
         .open(path)
         .map_err(ConfyError::OpenConfigurationFileError)?;
-    let s = toml::to_string_pretty(&cfg).map_err(ConfyError::SerializeTomlError)?;
+
+    let s;
+    #[cfg(feature = "toml_conf")] {
+        s = toml::to_string_pretty(&cfg).map_err(ConfyError::SerializeTomlError)?;
+    }
+   #[cfg(feature = "yaml_conf")] {
+        s = serde_yaml::to_string(&cfg).map_err(ConfyError::SerializeYamlError)?;
+    }
```

Comment:
This is a bit hard to read, IMO.

How about

```
let s = {
    #[cfg(feature = "toml_conf")] {
        toml::to_string_pretty(&cfg).map_err(ConfyError::SerializeTomlError)
    }
    #[cfg(feature = "yaml_conf")] {
        serde_yaml::to_string(&cfg).map_err(ConfyError::SerializeYamlError)
    }
}?;
```

(Not sure though whether this is actually better... :confused: )